### PR TITLE
Ft/contact page

### PR DESCRIPTION
**Pull request by gloriaumutoni**
Implemented a contact page that contains green text and two flexed parts with contact information and a disclaimer on the right-hand side. We have also made it responsive on mobile, tablet and large screens.

### CODE REVIEW

**Review by Chartine02 — APPROVED** [review]
Contact page implemented successfully.

**Review by Ntwalii — COMMENTED** [review]
On the client view, see the padding from the nav bar to the contact section

**Comment by josueahadi** [comment]
Use this link [https://deploy-preview-14--baumeisterr.netlify.app/](https://deploy-preview-14--baumeisterr.netlify.app/contact.html) to preview the contact page. I have noticed the deploy link above automatically redirects you to index.html. So far the feature from nav that links pages has not been merged with dev. TMI :)

**Review by beylar — APPROVED** [review]
Issue resolved.

File: src/sections/contact-section.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<div class="pt-[6%] container grid text-2xl grid-rows-2 md:grid-cols-2 gap-16 md:gap-20">
```
Comment (NoellaPromise) [inline src/sections/contact-section.html]: Is there a specific reason to use arbitrary value here?

File: src/sections/contact-section.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            <p class="md:text-3xl xl:text-5xl">10967 Berlin</p>
+        </div>
+
+        <div class="md:text-3xl xl:text-5xl">
+            <p>+49 (0)30 - 490 827 87</p>
+            <p>hallo@agentur-baumeister.com</p>
+        </div>
+
+        <div class="md:text-3xl xl:text-5xl">
+            <h2 class=" font-bold">UST-IDNR.</h2>
```
Comment (NoellaPromise) [inline src/sections/contact-section.html]: You may remove this unnecessary space in front of the `font-bold`  .

File: src/sections/contact-section.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+        <div>
+            <h2 class="md:text-3xl xl:text-5xl font-bold">REGISTERED OFFICE</h2>
+            <p class="md:text-3xl xl:text-5xl">Berlin, Deutschland</p>
+        </div>
+    </div>
+
+    <div class="flex flex-col md:justify-end md:items-end md:text-end md:text-xl xl:text-2xl md:gap-8">
+        <h2 class="font-bold">DISCLAIMER</h2>
+        <p class="pt-[3%] md:pt-0 2xl:w-3/5">
```
Comment (NoellaPromise) [inline src/sections/contact-section.html:25]: Same here , if there is any other way of styling this without using arbitrary value , you may use that way instead

**Review by NoellaPromise — COMMENTED** [review]
While styling and positioning shapes and text, try to minimize the use of arbitrary values.

**Review by NoellaPromise — COMMENTED** [review]
Also It is recommended to write the description on the PR. On this one there is no description

File: src/sections/contact-section.html (hunk 1 above)
Comment (josueahadi) [reply src/sections/contact-section.html]: @NoellaPromise Tailwind doesn't offer utility classes for small percentages like 6%, so we decided to use an arbitrary value `pt-[6%]` for it to respond on different screens.

**Review by NoellaPromise — APPROVED** [review]

**Pull request merged**